app/src/store/apiRequest.js

```diff
@@ -3,6 +3,11 @@ let apiGetMovies = url => {
         .then(res => res.json());
 }
 
+let apiGetMovie = url => {

```

Comment:
why does it have '...Movie' postfix everywhere? it can be reused everywhere)

Reply:
Fixed)